etc/configs/fake.json

```diff
@@ -7,10 +7,7 @@
         {
             "name": "arm1",
             "type": "arm",
-            "model": "fake",
```

Comment:
Will this pass tests? The tests that use `fake` rely on it having the actual fake mockup arm, right?

Reply:
to confirm I did cmf+shift+f on VScode. This allows me to search the entire RDK.
no tests use `etc/configs/fake.json`

Reply:
If the tests are happy with this change then I am too :)